### PR TITLE
Updated geometry parameters

### LINKED ISSUE #2192: Align zone volume to EnergyPlus model
This is for branch 1129.

### PR DESCRIPTION
**Pull request by milicag**
Closes #2192.

Once this is merged I can finalize the new 1502.

### CODE REVIEW

**Comment by AntoineGautier** [comment]
@milicag Do you need me to review this?

**Comment by milicag** [comment]
> @milicag Do you need me to review this?

Sure, I need one approval to merge.

File: Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo [hunk 1]
```diff
@@ -139,25 +139,25 @@ model Floor "Model of a floor of the building"
         iconTransformation(extent={{380,150},{400,170}})));
   Airflow.Multizone.DoorDiscretizedOpen opeSouCor(
     redeclare package Medium = Medium,
-    wOpe=10,
+    wOpe=10*(17.69 + 8.46)/(40.76 + 24.13),
```
Comment (mwetter) [inline Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo:142]: What is the rationale for setting `wOpe` to this expression, which evaluates for all four openings to 4.03 m width. This seems small, I think the dividing wall between core zone and the other four zones is much longer, which is why I set it to 10 m length.

**Review by mwetter — COMMENTED** [review]
See inline comments. Otherwise it looks good, assuming you took for `s` the aspect ratio of the building.

File: Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo (hunk 1 above)
Comment (milicag) [reply Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo:142]: I agree it is a bit small. The rationale is copied over from edits you made to the replaceable zone version, where you essentially scale the small office building sum of zone long and short wall with the sum of the same lengths for the larger building we used in the VAVReheat.

Comment (milicag) [reply Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo:142]: @mwetter Let me know what would be your preference.

Comment (mwetter) [reply Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo:142]: I suggest you make the openings about half the length of the wall, and add a comment so we know in a few years where the number came from.

Comment (milicag) [reply Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo:142]: Sounds good.

**Comment by milicag** [comment]
@mwetter Would you prefer to merge this one once it looks fine? I'm fine either way.

**Pull request merged**